src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,16 @@
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence"
+	version="2.0">
+	<persistence-unit name="interop-db">
```

Comment:
call interop-db-pu and use JTA

Reply:
we have a java standard main application so we used RESOURCE_LOCAL instead of JTA which is suggested for the container based application.